app/controllers/rapidfire/attempts_controller.rb

```diff
@@ -13,10 +14,14 @@ def new
     def create
       @attempt_builder = AttemptBuilder.new(attempt_params)
 
-      if @attempt_builder.save
-        redirect_to after_answer_path_for
-      else
-        render :new
+      respond_to do |format|
+        if @attempt_builder.save
+          format.html { redirect_to after_answer_path_for }
+          format.json { render json: @attempt_builder.to_model, include: :answers, status: 201 }
```

Comment:
The attempt builder was a little bit of a head-scratcher so I'm going to write up some docs on how to shape your JSON.